README.md

```diff
@@ -135,7 +135,7 @@ Alternatively you can point the CMake GUI tool to the CMakeLists.txt file and ge
 
```

Comment:
You could also argue about why it's better than executing `./libgit2_clar` directly. But yeah, I don't really mind